Orange/tests/test_statistics.py

```diff
@@ -126,6 +126,13 @@ def test_nanmin_nanmax(self):
                     nanmax(X_sparse, axis=axis),
                     np.nanmax(X, axis=axis))
 
+    def test_nansum(self):
+        for X in self.data:
```

Comment:
Can we use '@dense_sparse' decorator for running these tests? It also tests on sparse matrices with explicit zeros.